types/index.d.ts

```diff
@@ -747,7 +747,7 @@ export namespace Operation {
   ): xdr.Operation<RevokeClaimableBalanceSponsorship>;
 
   interface RevokeLiquidityPoolSponsorship extends BaseOperation<OperationType.RevokeSponsorship> {
-    balanceId: string;
+    liquidityPoolId: string;
```

Comment:
No test?

Reply:
Yeahh.. unfortunately the entire set of operations doesn't have tests. This is a one-off fix but I have a long list of how we can get better TypeScript testing in this library (and better API testing in general...). Unfortunately, it's a behemoth of a task.